pkg/strategy/grid2/strategy.go

```diff
@@ -1495,6 +1544,7 @@ func (s *Strategy) buildFilledPinOrderMapFromTrades(ctx context.Context, history
 		s.logger.Infof("[DEBUG] len of trades: %d", len(trades))
 
 		for _, trade := range trades {
+			s.logger.Infof("[DEBUG] %s", trade.String())
```

Comment:
remove this debugging message?

Reply:
I wanna keep this debug message for stage testing.